Solr/conf/dockets_schema.xml

```diff
@@ -0,0 +1,1295 @@
+             stored="true"
+             multiValued="false"/>
+      <field name="dateReargued"
+             type="date"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+      <field name="dateReargumentDenied"
+             type="date"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+      <field name="dateFiled"
+             type="date"
+             indexed="true"
+             stored="true"
+             multiValued="false"
+             docValues="true"/>
+      <field name="docketNumber"
+             type="string"
+             indexed="true"
+             stored="true"
+             multiValued="false"
+             termVectors="true"/>
+      <field name="caseName"
+             type="text_en_splitting_cl"
+             indexed="true"
+             stored="true"
+             multiValued="false"
+             termVectors="true"
+             termPositions="true"
+             termOffsets="true"/>
+      <field name="pacerCaseId"
+             type="int"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+      <field name="cause"
+             type="text_en_splitting_cl"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+      <field name="juryDemand"
+             type="text_en_splitting_cl"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+      <field name="jurisdictionType"
+             type="text_en_splitting_cl"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+
+      <!--DocketEntry-->
+      <field name="docketEntries"
+             type="text_en_splitting"
+             indexed="true"
+             stored="true"
+             multiValued="false"
+             termVectors="true"
+             termPositions="true"
+             termOffsets="true"/>
+
+      <!--Judge-->
+      <field name="assignedTo"
+             type="text_en_splitting_cl"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+
+      <!--Court-->
+      <field name="court"
+             type="text_en_splitting_cl"
+             indexed="true"
+             stored="true"
+             multiValued="false"/>
+
+      <!--special fields-->
+      <field name="_version_" type="long" indexed="true" stored="true"/>
+      <field name="django_ct" type="string" indexed="true" stored="true"
+             multiValued="false"/>
+      <field name="django_id" type="string" indexed="true" stored="true"
+             multiValued="false"/>
+
+      <!--copy fields-->
+      <field name="text" type="text_en_splitting"
+             indexed="true"
+             stored="true"
+             multiValued="true"
+             termVectors="true"
+             termPositions="true"
+             termOffsets="true"/>
+
+      <copyField source="court_id" dest="text" maxChars="30000"/>
```

Comment:
This looks like a better way to do the text than the way I've been doing it (putting it all in the django template). Is there a technical advantage to doing it this way? If so, we should get a bug filed to convert the other indexes to behave this way. Otherwise, I'd rather we keep them consistent.